src/components/Lend/LoanSearch/LoanSearchInterface.vue

```diff
@@ -6,7 +6,12 @@
 					Filter & Sort
 				</kv-button>
 
-				<kv-lightbox :visible="isLightboxVisible" variant="lightbox" @lightbox-closed="closeLightbox">
+				<kv-lightbox
+					:visible="isLightboxVisible"
+					variant="lightbox"
+					title=""
```

Comment:
Fine for now but not very "descriptive" for a screen reader. We should eventually add something like "Loan Filter Controls" or something of that nature...

Reply:
Ah I didn't realize this `title` was also used for `aria-label`. I found a workaround, passing `{{ null }}` to a slot template. Found here: https://github.com/vuejs/vue/issues/7064